src/common/components/ImageUpload/ImageUpload.vue

```diff
@@ -51,40 +62,45 @@ async function onUpdateImage (files: FileList) {
     onUpdate(files[0])
   }
 }
+const modelValue = defineModel<File | null>()
 </script>
 
 <template>
   <AvFileUpload
+    v-model:error="imageUpload.error.value"
+    v-model:valid-message="imageUpload.valid.value"
+    :file-upload-title="t('global.information.fileUpload.title')"
+    :file-upload-description="t('global.information.fileUpload.dragAndDrop')"
+    :delete-button-label="t('global.buttons.delete')"
+    :model-value="modelValue"
+    :file-name="defaultImage"
     :aria-describedby="describedBy"
-    :error="imageUpload.error.value"
-    :valid-message="imageUpload.valid.value"
     :accept="ACCEPTED_FILE_TYPES"
+    :on-delete-file="onDeleteImage"
+    @update:model-value="(value) => modelValue = value"
     @change="onUpdateImage"
     @on-drop-accept-type-error="() => { imageUpload.error.value = t('global.error.file.acceptType') }"
   >
-    <template #left>
+    <template
+      v-if="defaultImage || imageUpload.previewUrl.value"
+      #left
+    >
       <img
-        :src="imageUpload.previewUrl.value ?? defaultImage"
+        :src="defaultImageUrl ?? imageUpload.previewUrl.value!"
```

Comment:
```suggestion
        :src="imageUpload.previewUrl.value ?? defaultImageUrl"
```

Il faut que l'image que tu ajoutes prennes le pas sur celle par défaut non ?

Reply:
tu ne peut pas upload une image si un default image est déjà présent, il faut absolument supprimer ta photo avant de pouvoir upload une nouvelle

Reply:
Comment ça se passe quand il y a des photos placeholder @BanelhaqB ?